### DISCUSSION TITLE
Safe to update from 2023.2.2 to 2023.12.0?

### DISCUSSION THREAD

**Discussion by Sheugel** [music-assistant/support#1780] [Q&A] [answered]
Hello,
I've kept the 2023.2.2 version until now. Can I safely update to 2023.12.0?
Thanks

**Comment by OzGav** [comment]
Depends on what you mean by safely. What are you concerned about?

**Comment by Sheugel** [comment]
Just to make sure that it's going to work and that I won't have to manually uninstall it...

**Reply by OzGav** [reply, marked as answer]
I can’t say whether it will or it won’t as you have provided no details about your setup. All I can say is the server is still in beta, 1000s of people use it and we have outstanding issues which you can read on GitHub.
